src/EPR.RegulatorService.Facade.Core/Services/Regulator/RegulatorOrganisationService.cs

```diff
@@ -29,7 +31,9 @@ public class RegulatorOrganisationService : IRegulatorOrganisationService
 
         public async Task<CheckRegulatorOrganisationExistResponseModel?> GetRegulatorOrganisationByNation(string nation)
         {
-            var response = await _httpClient.GetAsync($"{_config.Endpoints.GetRegulator}{nation}");
+            string url = string.Format("{0}{1}", _config.Endpoints.GetRegulator, nation);
+
+            var response = await _httpClient.GetAsync(CheckURL(url));
```

Comment:
why not use that other pattern?
var uriBuilder = new UriBuilder(_httpClient.BaseAddress)
        {
            Path = url
        };

Reply:
Other pattern did not work.

Reply:
Please undo this change, ignore the SQ warning as peter is happy to leave it as is